bitbots_hcm/src/bitbots_hcm/hcm_dsd/decisions/decisions.py

```diff
@@ -246,9 +246,10 @@ def perform(self, reevaluate=False):
             return "ON_GROUND"
         # check if the robot is currently being picked up. foot have no connection to the ground,
         # but robot is more or less upright (to differentiate from falling)
-        if self.blackboard.pressure_sensors_installed and sum(self.blackboard.pressures) < 10 and \
-                abs(self.blackboard.smooth_accel[0]) < self.blackboard.pickup_accel_threshold and \
-                abs(self.blackboard.smooth_accel[1]) < self.blackboard.pickup_accel_threshold:
+        if self.blackboard.pressure_sensors_installed and not self.blackboard.simulation_active and \
+                sum(self.blackboard.pressures) < 10 and abs(self.blackboard.smooth_accel[0]) < \
+                self.blackboard.pickup_accel_threshold and abs(self.blackboard.smooth_accel[1]) < \
+                self.blackboard.pickup_accel_threshold:
```

Comment:
I think it would be more readable to not have line breaks in the individual conditions